ledger/src/blockstore.rs

```diff
@@ -87,8 +87,9 @@ pub struct Blockstore {
     data_shred_cf: LedgerColumn<cf::ShredData>,
     code_shred_cf: LedgerColumn<cf::ShredCode>,
     transaction_status_cf: LedgerColumn<cf::TransactionStatus>,
-    transaction_status_index_cf: LedgerColumn<cf::TransactionStatusIndex>,
-    active_transaction_status_index: RwLock<u64>,
+    address_signatures_cf: LedgerColumn<cf::AddressSignatures>,
```

Comment:
Are we going to have another "meta" data loss like we did from 1.0.13 -> 1.0.14 if an existing ledger is upgraded to the version that this PR ends up in, with the removal of `transaction_status_index_cf`?  This may be too hard to manage with our partners, so some backwards compat code may be needed to avoid losing the existing meta information 🤕

Reply:
This won't cause any "meta" data loss, but the pruning frequency could get off for one cycle, which might result in less data retained than expected, so, fair concern.
I think I just need to retain the transaction_status_index name here: https://github.com/solana-labs/solana/pull/9367/files#diff-5f6b1e8d3bf211c4eb9bc67a7718a665L42
But I'll probably just roll back the renaming commit for consistency.

Reply:
Ok cool.  I'll take another look then, please poke me when ready 👈

Reply:
Should be gtg now